artifactory.py

```diff
@@ -330,6 +350,28 @@ def encode_properties(parameters):
     return ";".join(result)
 
 
+# Declare contextlib class that was enabled in Py 3.7. Declare for compatibility with 3.5
```

Comment:
Leave a comment pls - where did you get this (just point to lines into .py file in python3.7 github source)